madrigal415/hmk-2/script.js

```diff
@@ -0,0 +1,114 @@
+// Myhubot Slack bot that does the following:
+
+
+/* Variables for random example */
+
+var prettyImages;
+prettyImages = ["http://www.apogeephoto.com/wp-content/uploads/2016/june2012/Noella_photos/3-A-CAOAK005copy.jpg", "http://www.jamesmatthewsphotography.com/jmp/wp-content/uploads/2014/05/5813-landscape-photography-merced-river-yosemite-california-usa.jpg", "http://www.orangejuiceblog.com/wp-content/uploads/2013/04/hb-bonfire.jpg", "http://www.grahamowengallery.com/photography/Flowers/3-21-09-California-Wildflowers.jpg"];
+
+// calling hubot
+module.exports = function(robot) {
+  
+    // Salutation
+    robot.respond(/hi|hello/i, function(msg) {
+        return msg.send("Hiyo!");
+    });
+
+    // Good morning 
+    robot.respond(/good morning sunshine, it's a (.*) day/i, function(res) {
+        var dayType = res.match[1];
+        var time = new Date();
+            var hours = time.getHours();
+          if (hours <12 ){
+            res.reply("I'm afraid it is no longer morning. You can still have a " + dayType + " day, but you better get moving.");
+          } else {
+            res.reply("Good morning to you, it IS a " + dayType + " day!");}
+    });
+
+
+    // Random example --- a pretty picture
+    robot.hear(/show me something pretty/i, function(msg) {
+        return msg.send(msg.random(prettyImages));
+    });    
+
+   // Friday
+   robot.respond(/is it friday/i, function(msg){
+        var today = new Date();
+        msg.reply(today.getDay() === 4 ? "Oh yea! Get ready for the weekend!" : "No. :(");
```

Comment:
Is it Friday? I don't think I've ever seen a ternary expression used in such a fun way. 😜 
If you feel comfortable using ternary expressions, definitely keep at it. Be wary of verbose ternary expressions; they quickly become difficult to read and fluster other developers.